src/app/styles/tailwind/ui/screens.js

```diff
@@ -0,0 +1,10 @@
+const screens = {
+  xs: '320px',
```

Comment:
Not use we need xs, tailwind use mobile first approach, e.g media query will be min-width: 320px, which doesn't make a lot of sense

Reply:
Got it! Thanks